projects/plugins/jetpack/tests/php/general/test-class.jetpack-gutenberg.php

```diff
@@ -13,12 +13,12 @@ public function set_up() {
 		parent::set_up();
 		if ( ! function_exists( 'register_block_type' ) ) {
 			$this->markTestSkipped( 'register_block_type not available' );
-			return;
+			return; // @phan-suppress-current-line PhanPluginUnreachableCode
```

Comment:
Probably shouldn't have the `return` at all here.

Reply:
🤷 Yeah, it's redundant, but I tend to give leeway to a `break` or `return` that makes code flow clearer to human readers.